manifold/src/boolean_result.cpp

```diff
@@ -674,6 +674,10 @@ Manifold::Impl Boolean3::Result(Manifold::OpType op) const {
   Timer simplify;
   simplify.Start();
 
+  outR.meshRelation_.allMeshID = inP_.meshRelation_.allMeshID;
+  outR.meshRelation_.allMeshID.insert(
+    inQ_.meshRelation_.allMeshID.begin(),
+    inQ_.meshRelation_.allMeshID.end());
```

Comment:
Some of these meshIDs might not be referenced anymore, right? I take it that doesn't matter since this is an internal detail?

Reply:
Yes, the cache is meant to be an over approximation of the set of mesh ids. IDs that are not being referenced will be removed when we duplicate the impl.